modules/openapi-generator/src/main/resources/aspnetcore/3.0/wwwroot/web.config

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please delete this file.

Reply:
Should I remove the entire www directory as that's not part of the template anymore?

Reply:
Yes. It should be fine to remove it.